airflow/providers/common/sql/doc/adr/0002-return-common-data-structure-from-dbapihook-derived-hooks.md

```diff
@@ -0,0 +1,139 @@
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+ -->
+
+# 2. Return common data structure from DBApiHook derived hooks
```

Comment:
This should be point number 1 instead?

Reply:
Nope. It's 2.  1st ADR is about accepting ADRs. This is part of the title not a numbered list.

Reply:
Maybe this should be something like ADR-2 to avoid confusion?

Reply:
It's automatically generated like that by the ADR tool (see ADR-01 - it has link to the tool. https://github.com/npryce/adr-tools ), so I'd prefer to leave it as is. If we have new ADRs genarated with 

```
adr new "Some title of the next ADR"
``` 

it will get the right file name + basic content with this format, so I prefer to leave it as is.

Reply:
Looks like it’s possible to customise the template but the tool does not provide a good config format. Honestly the entire project does not feel very actively developed.

Reply:
Correct. The whole idea is to have it as lightweight as possible and generally noone expected any changes after it has been created. It's the kind of super-simple tools that are done once and should serve their purpose for - basically - for multiple years without any changes (similar case as with json specification that was released once).

Reply:
Also one thing here. While ADR tool exists and is good to bootstrap things, it's also fine to edit the ADR records manually - those are simply inter-linked (sometimes) markdown documents that follow the numbering scheme so that we can refer to them in the context of the part of teh software they relate to. For breeze I thin 90% of the ADRs I just added by hand. So the tool is really to get you off the ground quickly - mostly to make the first commit (but still it's good to keep the convention the tool uses).